colour/models/rgb/transfer_functions/red_log.py

```diff
@@ -212,16 +220,21 @@ def log_encoding_Log3G10(x):
 
     Examples
     --------
-    >>> log_encoding_Log3G10(0.18)  # doctest: +ELLIPSIS
+    >>> log_encoding_Log3G10(0.18, legacy_curve=True)  # doctest: +ELLIPSIS
     0.3333336...
+    >>> log_encoding_Log3G10(0.0)  # doctest: +ELLIPSIS
+    0.0915514...
     """
 
     x = np.asarray(x)
 
-    return np.sign(x) * 0.222497 * np.log10((np.abs(x) * 169.379333) + 1)
+    if legacy_curve:
```

Comment:
Wondering if we should have a well defined `if ... else ...` block here. I'm nitpicking really :)